src/components/Overlay/Overlay.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import './Overlay.scss';
+
+interface Props {
+  className: string;
+}
+
```

Comment:
Can you change this to

```tsx
export type OverlayProps = React.HTMLAttributes<HTMLDivElement>;
```

You want the type to always be exported. If you use the scaffold `yarn scaffold:component Overlay` it would autogenerate all this for you, including spec/stories.